src/client/scripts/cardSubmitModal.js

```diff
@@ -22,37 +23,18 @@ Template.cardSubmitModal.events({
 
     "submit #card": function(eve){
         eve.preventDefault();
-        var author = Meteor.user() ?
-                        Meteor.user().profile.name :
-                        Session.get("author");
-        var thought = eve.target.thoughts.value;
+        var cardData = getCardData(eve);
         var tags = eve.target.tags.value;
-
-        var category = eve.target.categoryDropdown.value;
-
-        if(category === "Select a Category") {
-            SnackbarMethods
-                .DisplayMessage("Enter a category for your thought", 3000);
-            return ;
-        }
-        if(thought.length == 0) {
-            SnackbarMethods.DisplayMessage("Enter a thought", 3000);
-            return ;
-        }
-        if(!author) {
-            SnackbarMethods.DisplayMessage("Please set alias or sign in", 3000);
-            return;
-        }
+        var cat = 0, tex = 1, auth = 2;
```

Comment:
I dont like this,  if I make changes in getCardData I can really screw this up without knowing.